CRM/Contact/Import/Parser/Contact.php

```diff
@@ -1555,6 +1555,401 @@ public function createContact(&$formatted, &$contactFields, $onDuplicate, $conta
     return $newContact;
   }
 
+  /**
+   * Legacy format profile contact parameters.
+   *
+   * This is a formerly shared function - most of the stuff in it probably does
+   * nothing but copied here to star unravelling that...
```

Comment:
star -> start
:)

Reply:
Hmm - I might take that on board for a follow up as I have a few patches locally that build on this